test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs

```diff
@@ -20,14 +20,14 @@ public class GivenThatIWantANewAppWithSpecifiedType : TestBase
         [InlineData("C#", "classlib", false, false)]
         [InlineData("C#", "mstest", false, false)]
         [InlineData("C#", "xunit", false, false)]
-        [InlineData("C#", "web", false, false)]
-        [InlineData("C#", "mvc", false, false)]
-        [InlineData("C#", "webapi", false, false)]
+        [InlineData("C#", "web", true, false)]
+        [InlineData("C#", "mvc", true, false)]
+        [InlineData("C#", "webapi", true, false)]
         [InlineData("C#", "angular", false, true)]
         [InlineData("C#", "react", false, true)]
         [InlineData("C#", "reactredux", false, true)]
         [InlineData("F#", "console", false, false)]
-        //  https://github.com/dotnet/cli/issues/7476
+        // issue with netstandard2.0 --- formerly was issue with: https://github.com/dotnet/cli/issues/7476
```

Comment:
We should linke to a new issue then that describes more detail than issue with netstandard2.0

Reply:
@nguerrera - sounds good. I can make the appropriate changes in the code, but I'm not certain what to make for an issue. The problem I'm seeing from the test results (and from manual testing) is that I can create & restore the relevant templates, but the restore seems incorrect. The F# classlib template won't build, the error being:
```c:\temp\Creation\scratch\F#_Classlib\unknown(1,1): error FS0078: Unable to find the file 'netstandard.dll' in any of C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework c:\temp\Creation\scratch\F#_Classlib C:\Program Files\dotnet\sdk\2.1.0-preview1-007099\FSharp\ [c:\temp\Creation\scratch\F#_Classlib\F#_Classlib.fsproj]```
... which is strange, since it apparently restored correctly. 

But the second test I left un-enabled uses the C# classlib template, which does build. The problem with that test is something different, but the underlying cause may be the same. For projects targeting netstandard2.0, when I run a command like:
`dotnet restore --configfile c:\GitHub\cli\NuGet.Config --packages packages`the restore process reports success, but it doesn't create the requested packages directory.

Reply:
Just putting the comment above in a CLI bug that says to investigate and re-enable when fixed is fine.